packages/components/src/Tabs/Tabs.tsx

```diff
@@ -87,15 +89,28 @@ export function Tab({ label }: TabProps) {
 interface InternalTabProps {
   readonly label: string;
   readonly selected: boolean;
-  onClick(): void;
+  activateTab(): void;
+  onClick?(): void;
 }
 
-export function InternalTab({ label, selected, onClick }: InternalTabProps) {
+export function InternalTab({
+  label,
+  selected,
+  activateTab,
+  onClick = () => undefined,
```

Comment:
Does this work here instead?

```suggestion
  onClick = () => return,
```

That way we are returning `void` instead of a value (even if that value is undefined).

Reply:
yep it works! Changes pushed